src/inmanta/server/services/resourceservice.py

```diff
@@ -539,10 +540,15 @@ async def resource_deploy_done(
                 if not keep_increment_cache:
                     self.clear_env_cache(env)
 
+                extra_fields = {}
+                if status == ResourceState.deployed:
+                    extra_fields["last_success"] = finished
```

Comment:
If we use `finished` here, don't we risk missing events between when `get_resource_events` was called and when `resource_deploy_done` is called? I'd think we either need to use `started` (meaning we also need to revert the change you made to the filter higher up) or allow the handler to explicitly register the time at which it looked at events (which would also require an update of the filter).